samples/managed-identity/.vscode/launch.json

```diff
@@ -20,7 +20,7 @@
             "OS-COMMENT5": "Enable launching a web browser when ASP.NET Core starts. For more information: https://aka.ms/VSCode-CS-LaunchJson-WebBrowser",
             "env": {
                 "ASPNETCORE_ENVIRONMENT": "Development",
-                "ASPNETCORE_URLS": "http://+:80"
+                "ASPNETCORE_URLS": "http://+:5000"
```

Comment:
why are we changing the port here?

Reply:
I shouldn't have committed this file, let me revert it. Thanks for catching this.

Reply:
Reverted.